tests/utils.go

```diff
@@ -3316,6 +3316,13 @@ func RemoveHostDiskImage(diskPath string, nodeName string) {
 	Eventually(getStatus, 30, 1).Should(Equal(k8sv1.PodSucceeded))
 }
 
+// Skip tests that use ISCSI PVC's with IPv6 until https://github.com/kubevirt/kubevirt/issues/4150 solved
```

Comment:
i would add this comment to the Skip
see please how we are doing this on all the other skippers

Reply:
+1; having the issue on the `Skip`s body is helpful.

Reply:
Done